libs/sqlite3/Makefile

```diff
@@ -67,11 +75,20 @@ $(call Package/sqlite3/Default/description)
  formats.
 endef
 
+
 TARGET_CFLAGS += $(FPIC) \
 	-DSQLITE_ENABLE_UNLOCK_NOTIFY=1 \
 	-DHAVE_ISNAN=1 \
 	-DHAVE_MALLOC_USABLE_SIZE=1
 
+ifneq ($(CONFIG_SQLITE_FTS3),y)
+TARGET_CFLAGS += -USQLITE_ENABLE_FTS3
```

Comment:
-U? does this actually work?

Reply:
never mind, had never seen the undef flag before, would it not be more standard to define =0 ?

Reply:
I honestly have no idea

Reply:
Yes this works see Preprocessor options at https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html